src/main/kotlin/no/nav/helsearbeidsgiver/dokarkiv/DokArkivClient.kt

```diff
@@ -72,25 +71,15 @@ class DokArkivClient(
                     logger.error("Journalpost ble opprettet, men ikke ferdigstilt. journalpostId=[${it.journalpostId}] $idFragment")
                 }
             }
-            .onFailure {
-                if (it.instanceOf(ClientRequestException::class)) {
-                    it as ClientRequestException
-                    if (it.response.status == HttpStatusCode.Conflict) {
-                        val journalpost = runBlocking { it.response.body<OpprettOgFerdigstillResponse>() }
-                        if (!journalpost.journalpostId.isNullOrEmpty()) {
-                            logger.info(
-                                "Fikk 409 Conflict ved journalføring med referanse(id=${request.eksternReferanseId}) og tittel (${request.tittel}. " +
-                                    "Bruker eksisterende journalpostId (${journalpost.journalpostId}) fra respons.",
-                            )
-                            return journalpost
```

Comment:
Flytter litt rundt på ting for å unngå denne, siden den skipper resten av pipen. Generelt så kan det være skummelt å bruke `return` i pipe av denne grunnen.